platform/linuxbsd/key_mapping_x11.cpp

```diff
@@ -323,13 +323,13 @@ Key KeyMappingX11::get_keycode(KeySym p_keysym) {
 		}
 	}
 
-	return KEY_NONE;
+	return Key::NONE;
 }
 
 KeySym KeyMappingX11::get_keysym(Key p_code) {
 	// kinda bruteforce.. could optimize.
 
-	if (p_code < 0x100) { // Latin 1, maps 1-1
+	if (p_code < Key::END_LATIN) { // Latin 1, maps 1-1
```

Comment:
I added this constant as an enum member.

Reply:
Should it be `END_LATIN1`?

Reply:
Yes, I think that's an improvement.